cmd/kubeadm/app/api/install/install.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
please make the path to this file (and other api machinery files) be `.../apis/<your group name>/...`, not `/api`